bedrock/firefox/views.py

```diff
@@ -611,6 +618,8 @@ def get_template_names(self):
                 template = 'firefox/developer/whatsnew.html'
             else:
                 template = 'firefox/whatsnew/index.html'
+        elif version.startswith('92.') and locale.startswith('en-'):
+            template = 'firefox/whatsnew/whatsnew-fx92-en.html'
```

Comment:
I'm still seeing this template when I veiw http://localhost:8000/en-US/firefox/92.0/whatsnew/china/, so you might need to add an exception still?